.github/workflows/packages-e2e-tests.yaml

```
@@ -37,19 +41,71 @@ jobs:
         id: tag
         run: echo "tag=$(make version)" >> $GITHUB_OUTPUT
 
-      # Logging into Docker Hub makes the job less flaky, but it relies on repo secrets so
-      # does not work for external contributors. Let's compromise and do this for internal
-      # PRs at least (for now). We can always improve the situation later.
-      - name: Login to Docker Hub
-        if: ${{ github.event.pull_request.head.repo.full_name == 'cilium/tetragon' }}
-        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3.0.0
-        with:
-          username: ${{ secrets.DOCKER_HUB_USERNAME_CI }}
-          password: ${{ secrets.DOCKER_HUB_PASSWORD_CI }}
-
       - name: Generate Tetragon Tarball
         id: tetragon-tarball
-        run: make tarball
+        run: |
+          make tarball
+          mkdir ${{ matrix.upload_path }}
+          mv ./build/${{ matrix.arch }}/linux-tarball/tetragon-${{ steps.tag.outputs.tag }}-${{ matrix.arch }}.tar.gz ./${{ matrix.upload_path }}
+        if: ${{ matrix.cross_compile == 'no' }}
+
+      - name: Generate Cross Compiled Tetragon EE Tarball
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y qemu qemu-user-static binfmt-support
+          sudo update-binfmts --display
+          make TARGET_ARCH=${{ matrix.arch }} tarball
+          mkdir ${{ matrix.upload_path }}
+          mv ./build/${{ matrix.arch }}/linux-tarball/tetragon-${{ steps.tag.outputs.tag }}-${{ matrix.arch }}.tar.gz ./${{ matrix.upload_path }}
+        if: ${{ matrix.cross_compile == 'yes' }}
+
+      # Cache tarball releases for later
+      - name: Save tetragon-${{ steps.tag.outputs.tag }}-${{ matrix.arch }}.tar.gz Tarball
+        uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
+        with:
+          name: tetragon-${{ steps.tag.outputs.tag }}-${{ matrix.arch }}
+          path: ${{ matrix.upload_path }}
+          retention-days: 1
+
+  standalone-tarball-tests:
+    needs: [standalone-tarball-builds]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-22.04
+            arch: amd64
+            match_arch: x86-64
+            cross_compile: no
+            upload_path: upload/
+          - os: buildjet-2vcpu-ubuntu-2204-arm
```

please remove buildjet and use actuated instead for arm runners (those are free, and we pay for buildjet). You can adjust the number of CPU and amount of RAM independently as needed, but here is a suggestion:
```suggestion
          - os: actuated-arm64-4cpu-8gb
```

---

Done, thank you!